en/docs/references/connect-with-protected-third-party-applications.md

```diff
@@ -0,0 +1,8 @@
+# Connect with Protected Third Party Applications
+
+When your Choreo component connects with protected third-party applications, the firewalls of these applications might block the requests of the Choreo component. To address this, you can whitelist the following public IP ranges of Choreo that it assigns to Choreo components as client IPs:
```

Comment:
We would need to decide on an alternative term to whitelist because we decided to avoid terms that can be racially sensitive.